src/Domain/LeanCode.CQRS.Execution/CommandExecutor.cs

```diff
@@ -29,7 +29,11 @@ public async Task<CommandResult> RunAsync(TAppContext appContext, ICommand comma
             try
             {
                 var res = await executor.ExecuteAsync(appContext, command);
-                logger.Information("Command {@Command} executed successfully", command);
+                if (res.WasSuccessful)
```

Comment:
Maybe some meaningful log for case when result is not successful? :thinking:

Reply:
If the result is not succesful it means it failed validation, and there is a warning in the `ValidationElement` in the pipeline.